.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        driver_version: ["470.82.01", "510.47.03", "515.65.01", "525.85.12", "535.54.03"]
+        driver_version: ["470.82.01", "510.47.03", "515.65.01", "525.85.12", "535.54.03", "550.54.15"]
```

Comment:
After testing, we can remove the older driver version

Reply:
can we add an explicit TODO?

Reply:
is the "older driver version" referring to the 470 driver? We would need to update the driver selection logic too in that case right? Might want to note that in the TODO as well.